### DISCUSSION TITLE
Saturday Meeting

### DISCUSSION THREAD

**Discussion by NathanGrenier** [NathanGrenier/SOEN-341#65] [Polls]
The meeting is currently scheduled for 3pm. Feel free to comment if this time isn't convenient for you.

**Comment by briantkatch** [comment]
Prefer to meet during the week if possible (possibly immediately before/after the lab/lecture) but Saturday will work if nothing else.

**Reply by NathanGrenier** [reply]
Pretty sure we will also have a meeting during the lab as their meant for teams to collaborate in person. This meeting on Saturday is just to decide if we'll use SvelteKit and to tie up any other loose ends before the semester starts.
